### DISCUSSION TITLE
Import project from Cloud9 to Github

### DISCUSSION THREAD

**Discussion by TheOuen** [community/community#23897] [New to GitHub]
<p>I’m having problems importing a project to Github.</p>
<p>The error says the following and I can’t go further - “Your old project requires credentials for <strong>read-only access</strong> . We will only temporarily store them for importing.”</p>
<p>I kind of don’t understand what the next steps should be. If someone could help me out in laymen’s terms that would be appreciated</p>

**Comment by sr229** [comment, marked as answer]
<p>Alternatively, you can just make a Remote like so:</p>
<pre><code class="lang-bash">$ git remote add github https://github.com/&lt;project&gt;
</code></pre>
<p>Then push to the remote, this should save you the headache with their built-in import tools for now.</p>
